packages/p2p-v2/src/pages/my-ads/screens/MyAds/MyAdsTable/MyAdsTable.scss

```diff
@@ -0,0 +1,15 @@
+.p2p-v2-my-ads-table {
+    width: 100%;
+    height: 100%;
+    position: relative;
+    & .p2p-v2-table {
+        &__header {
+            // stylelint-disable-next-line declaration-no-important
+            grid-template-columns: repeat(3, 1.6fr) 1.9fr 3fr 1.9fr !important;
```

Comment:
can we avoid using `!important`?

Reply:
had to add important because inline styles was added in tablecomponent as repeat(num of columns, 1).. or let me just remove the inline styles..